app/controllers/api/v2/compliance/policies_controller.rb

```diff
@@ -90,9 +90,13 @@ def content
 
       def tailoring
         @tailoring_file = @policy.tailoring_file
-        send_data @tailoring_file.scap_file,
-                  :type => 'application/xml',
-                  :filename => @tailoring_file.original_filename
+        if @tailoring_file
+          send_data @tailoring_file.scap_file,
+                    :type => 'application/xml',
+                    :filename => @tailoring_file.original_filename
+        else
+          render(:json => { :error => { :message => "No Tailoring file assigned for policy with id #{@policy.id}" } }, :status => 404)
```

Comment:
since this is no longer internal server error but rather expected response, I think the text should be extracted for translations